torchao/float8/config.py

```diff
@@ -146,6 +146,13 @@ class Float8GemmConfig:
     use_fast_accum: bool = False
 
 
+# Pre-made recipes for common configurations
+class Float8LinearRecipeName(enum.Enum):
+    TENSORWISE = "tensorwise"
+    ROWWISE = "rowwise"
+    ROWWISE_WITH_GW_HP = "rowwise_with_gw_hp"
```

Comment:
nit: wondering if gw_hp should be clarified somewhere as far as what it means?
it might also make sense to allow a fully written out version